frontend/public/components/_sysevent-stream.scss

```diff
@@ -82,17 +82,19 @@ $color-dark-border: #ddd;
   flex: none;
 }
 
-.co-sysevent__timestamp {
-  display: inline-block;
+.co-sysevent__details {
+  display: flex;
+  justify-content: space-between;
 }
 
-.co-sysevent__header {
-  justify-content: space-between;
+.co-sysevent__timestamp__message {
```

Comment:
BEM says [not to nest elements](http://getbem.com/faq/#css-nested-elements), so this should probably be something like `.co-sysevent__message`.